recipes/openmeeg/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set version = "2.4.0.dev" %}
+
+package:
+  name: openmeeg
+  version: {{ version }}
+
+source:
+  url: https://github.com/openmeeg/openmeeg/archive/2.4-rc2.tar.gz
+  sha256: 254eb82717ffec8b79618c281fe5c87734b2f8d804ef369d414d91b1b71cd163
+
+build:
+  number: 0
+  skip: true  # [win32]
+  msvc_compiler: 14.0
+  features:
+    - vc14  # [win]
+
+requirements:
+  build:
+    - cmake >=3.8
+    - toolchain
+    - python
+    - numpy
+    - swig
+    - hdf5 1.8.18|1.8.18.*
+    - zlib 1.2.*
+    - libmatio >=1.5.12
+    - clangdev 4.0*  # [unix]
+    - libcxx 4.0.*  # [osx]
+    - openmp 4.0.*  # [osx]
+    - libgfortran  # [unix]
+    - openblas 0.2.20|0.2.20.*
+
+  run:
+    - python
+    - numpy
+    - swig
+    - hdf5 1.8.18|1.8.18.*
+    - zlib 1.2.*
+    - libmatio >=1.5.12
+    - clangdev 4.0*  # [osx]
+    - libcxx 4.0.*  # [osx]
+    - openmp 4.0.*  # [osx]
+    - libgfortran  # [osx]
+    - openblas 0.2.20|0.2.20.*
+
+test:
+  imports:
+    - openmeeg  # [unix]
```

Comment:
Does this fails on mac / windows?

Reply:
yes, it does fail in windows.

Reply:
The package is skipped in win32 and build for win64. However, the python bindings are not supported in windows. But we do distribute openmeeg through conda.